src/map/skill.c

```diff
@@ -7471,7 +7471,7 @@ int skill_castend_nodamage_id(struct block_list *src, struct block_list *bl, uin
 		case NPC_LICK:
 			status_zap(bl, 0, 100);
 			clif->skill_nodamage(src,bl,skill_id,skill_lv,
-				sc_start(src,bl,type,(skill_lv*5),skill_lv,skill->get_time2(skill_id,skill_lv)));
+				sc_start(src,bl,type,(skill_lv*20),skill_lv,skill->get_time2(skill_id,skill_lv)));
```

Comment:
Code Styling -
Space after any comma between two arguments of a function. eg. `func(src, bl)`
Space before and after a binary operator.  eg. `a * b`